conda/recipes/rmm/meta.yaml

```diff
@@ -20,7 +20,7 @@ build:
 
 requirements:
   build:
-    - cmake >=3.14.0
+    - cmake >=3.18.0
```

Comment:
I don't think cmake is even required for the Python build, but we should update the `librmm` recipe as well.

Reply:
Changed it in `librmm` recipe, should we just remove it from `rmm` build recipe?

Reply:
Yea, if we could remove it from the `rmm` recipe that would be nice.

Reply:
Done